app/src/main/res/values/strings.xml

```diff
@@ -1473,6 +1473,15 @@ Alternatively, you can leave a note (with a photo)."</string>
     <string name="quest_tracktype_grade4">Mostly soft</string>
     <string name="quest_tracktype_grade5">Soft</string>
 
+    <string name="quest_entrance_reference">What is signed at this entrance?</string>
+    <string name="quest_entrance_reference_select_flat_range_and_code">Code of entrance and range of flats</string>
+    <string name="quest_entrance_reference_select_flat_range_only">Range of flats</string>
+    <string name="quest_entrance_reference_select_code_only">Code of entrance</string>
+    <string name="quest_entrance_reference_nothing_signed">No sign with code or flat range</string>
+    <string name="quest_entrance_reference_reference_label">Code of entrance:</string>
```

Comment:
Sure this and others should not be "entrance code"?

Reply:
What would you prefer?

Reply:
If this about a code that denominates an entrance, then "entrance code" I do prefer.